spec/lib/govuk/markdown_renderer_spec.rb

```diff
@@ -0,0 +1,180 @@
+require 'rails_helper'
+
+# Ignore whitespace
+def expect_equal_ignoring_ws(first, second)
+  expect(first.lines.map(&:strip).join('')).to eq(second.lines.map(&:strip).join(''))
+end
+
+describe Govuk::MarkdownRenderer do
+  let(:html) { Govuk::MarkdownRenderer.render(markdown) }
+
+  context 'ul' do
+    let(:markdown) do
+      <<~MD
+        - item
+        - item
+        - item
+      MD
```

Comment:
The specific example of a list we saw today was a list that followed a paragraph with no line-break. Can we test that we render that correctly here too?

Reply:
When I spoke to @dankmitchell he said he would look at this in a follow-up PR.